agent/engine/docker_task_engine.go

```diff
@@ -1192,7 +1192,19 @@ func (engine *DockerTaskEngine) startContainer(task *apitask.Task, container *ap
 func (engine *DockerTaskEngine) provisionContainerResources(task *apitask.Task, container *apicontainer.Container) dockerapi.DockerContainerMetadata {
 	seelog.Infof("Task engine [%s]: setting up container resources for container [%s]",
 		task.Arn, container.Name)
-	cniConfig, err := engine.buildCNIConfigFromTaskContainer(task, container, true)
+	containerInspectOutput, err := engine.inspectContainerByName(task.Arn, container.Name)
+	if err != nil {
+		return dockerapi.DockerContainerMetadata{
+			Error: ContainerNetworkingError{
+				fromError: errors.Wrap(err,
+					"container resource provisioning: unable to inspect pause container"),
+			},
+		}
+	}
+
+	task.SetPausePIDInVolumeResource(strconv.Itoa(containerInspectOutput.State.Pid))
```

Comment:
Does this send it to all volume resources, or just one? Maybe reword the method if it affects all of them.

Reply:
i wrongly assumed there can only be one volume resource per task. renamed method to SetPausePIDInVolumeResources and updated to send it to all volume resources